kernel/aster-nix/src/net/socket/options/mod.rs

```diff
@@ -17,7 +17,11 @@ impl_socket_options!(
     pub struct ReusePort(bool);
     pub struct SendBuf(u32);
     pub struct RecvBuf(u32);
+    pub struct RcvTimeoOld(timeval_t);
+    pub struct SndTimeoOld(timeval_t);
+    pub struct TimestampOld(bool);
```

Comment:
Avoids using C type like `timeval_t` in `net` module. C types are supposed to only be used in `syscall` module and should be converted to Rust type when calling functions in submodules.

So these two structs should be defined as
```rust
pub struct RcvTimeoOld(Duration);
pub struct SndTimeoOld(Duration);
```